projects/plugins/jetpack/functions.photon.php

```diff
@@ -305,24 +305,6 @@ function jetpack_photon_url_scheme( $url, $scheme ) {
 	return preg_replace( '#^([a-z:]+)?//#i', $scheme_slashes, $url );
 }
 
-/**
- * A wrapper for PHP's parse_url, prepending assumed scheme for network path
- * URLs. PHP versions 5.4.6 and earlier do not correctly parse without scheme.
- *
- * WP ships with a wrapper for parse_url, wp_parse_url, that should be used instead.
- *
- * @see https://php.net/manual/en/function.parse-url.php#refsect1-function.parse-url-changelog
- * @deprecated 7.8.0 Use wp_parse_url instead.
- *
- * @param string  $url The URL to parse.
- * @param integer $component Retrieve specific URL component.
- * @return mixed Result of parse_url
- */
-function jetpack_photon_parse_url( $url, $component = -1 ) {
```

Comment:
This function is used in the following w.org plugin:

- [ ] `widget-for-eventbrite-api`: frontend/class-frontend.php,65

Reply:
`widget-for-eventbrite-api` is under active development (13hr ago last update) and 2k installs. Function is not gated ([#](https://plugins.trac.wordpress.org/browser/widget-for-eventbrite-api/trunk/frontend/class-frontend.php#L62)).

The function can be wholesale replaced with `wp_parse_url` and their min WP version works. Opened a support ticket: https://wordpress.org/support/topic/update-jetpack-function/#new-topic-0

Suggest we leave this in for now with adding an internal note to remove it a fair amount of time after they've updated their plugin.